Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:2.5.1
```

Comment:
@JaroslawSokol Please try with slim image

Reply:
At first step we are going to build images with current versions and do not exchange and upgrades base images.
As a second step we have all apps builded from master - we will have optimization process changing base images, upgrades to new versions, etc.